app/src/main/java/com/example/cm/data/models/MeetupRequest.java

```diff
@@ -5,23 +5,18 @@
 import androidx.annotation.NonNull;
 
 import com.google.firebase.firestore.Exclude;
-import com.google.android.gms.maps.model.LatLng;
 
 import java.util.Calendar;
 import java.util.Date;
 import java.util.GregorianCalendar;
-import java.util.concurrent.TimeUnit;
 
 public class MeetupRequest extends Request {
 
     private String meetupId;
-    private String location;
     private String imageUrl;
     private Date meetupAt;
```

Comment:
Remove meetupAt since that's an information that can be read from the corresponding Meetup Model